### PR TITLE
Exosuit weapon refactor

### PR DESCRIPTION
**Pull request by Ben10083**
Exosuit laser now considered heavy, damage set to 35 (10 increase from old, 10 less than usual heavy laser)
Exosuit smg now can switch to 3-shot burst with alt click
Exosuit smg ammo count increased to 30 shots.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
The exosuit SMG can already swap to three and five round burst by using the firemode command.

**Comment by Ben10083** [comment]
> The exosuit SMG can already swap to three and five round burst by using the firemode command.

With the relation of exosuit and wearer, I don't think that is intended.

File: code/modules/heavy_vehicle/equipment/combat.dm [hunk 1]
```diff
@@ -76,11 +81,18 @@
 	self_recharge = TRUE
 	has_safety = FALSE
 
+/obj/item/gun/energy/mountedsmg/mech
+	max_shots = 30
+	firemodes = list(
+		list(mode_name = "semi-automatic", burst = 1, fire_delay = 0,    move_delay = null, burst_accuracy = null,                dispersion=list(0)),
+		list(mode_name = "3-round burst",  burst = 3, fire_delay = null, move_delay = 4,    burst_accuracy = list(0,-1,-1),       dispersion=list(0, 15, 15)),
```
Comment (Alberyk) [inline code/modules/heavy_vehicle/equipment/combat.dm:88]:
```suggestion
		list(mode_name = "3-round burst",  burst = 3, fire_delay = null, move_delay = 4,    burst_accuracy = list(0,-1,-1),       dispersion=list(0, 15, 15))
```

**Review by Alberyk — CHANGES_REQUESTED** [review]

**Review by Alberyk — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**